src/users/useCases/createUser/CreateUserDTO.ts

```diff
@@ -0,0 +1,3 @@
+export interface CreateUserDTO {
```

Comment:
│   README.md
│   package.json  
│   ...   
│
└─── src
│   │   index.ts // 5. Framework and Drivers
│   │   server.ts // start master and forks other workers
│   │   config.ts // get config from env or config file
│   │   database.ts
│   │   logger.ts
│   │
│   └─── users
│   │   │   api
│   │   │   │route
│   │   │   │controller
│   │   │   │dto 
│   └─── use-cases
│   │   │   │index.js
│   │   │   │createUser.js
│   │   │   │updateUser.js    
│   └─── repository
│   │   │   │index.js
│   │   │   │createUser.js
│   │   │   │updateUser.js    
│   └─── entity
│   │   └─── users
│   │   │   │   index.js
│   │   │   │   create-user.js
│   │   │   │   ...
│   │   └─── ...
└─── ...

Reply:
so we will create "createUser", "updateUser" entities (not only User entity)?

Reply:
yes

Reply:
but I think User entity should have an ID => what is the ID for createUser, updateUser entity?

Reply:
export class UserCreateDTO {
    username: string;
    email: string;
    password: string;

    constructor(username: string, email: string, password: string) {
        this.username = username;
        this.email = email;
        this.password = password;
    }
}

export class UserUpdateDTO {
    id: string; // uuid
    username: string;
    email: string;

    constructor(id: string, username: string, email: string) {
        this.id = id;
        this.username = username;
        this.email = email;
    }
}

Reply:
yeah, it is for DTO. But for entity, we will create just 1 entity User right?

Reply:
entity should have everything